assets/js/modules/pagespeed-insights/components/common/LabReportMetrics.js

```diff
@@ -103,6 +129,12 @@ export default function LabReportMetrics( { data, error } ) {
 			</table>
 			<div className="googlesitekit-pagespeed-report__row googlesitekit-pagespeed-report__row--last">
 				<ReportDetailsLink />
+				<Button
+					href="#update"
```

Comment:
Why the `href` here? If we just want a normal `<button>` that won't submit, we can use `type="button"`.

Reply:
Argh, that was a mistake, thanks! Didn't mean to commit it; non-`href` should be fine 👍🏻